src/v/cluster/ntp_callbacks.h

```diff
@@ -75,22 +75,22 @@ class ntp_callbacks {
       model::partition_id part,
       Args&&... args) const {
```

Comment:
would it be enough to use a const ref then ?

Reply:
Good idea. Yes, I think so, let me make sure.